Marlin/Marlin_main.cpp

```diff
@@ -4101,10 +4118,24 @@ inline void gcode_M104() {
         setTargetHotend1(temp == 0.0 ? 0.0 : temp + duplicate_extruder_temp_offset);
     #endif
 
+    /**
+     * We use halve EXTRUDE_MINTEMP here to allow nozzles to be put into hot
+     * stand by mode, for instance in a dual extruder setup, without affecting
+     * the running print timer.
+     */
+    if (temp <= (EXTRUDE_MINTEMP/2)) {
```

Comment:
This should probably be `if (temp <= (EXTRUDE_MINTEMP) / 2)` instead. The parentheses around settings is to handle cases like `#define EXTRUDE_MINTEMP 170 - 20` where someone forgot to add their own parentheses.